src/harness/fourslashImpl.ts

```diff
@@ -4504,8 +4504,8 @@ namespace FourSlash {
         };
     }
 
-    function stringify(data: any, replacer?: (key: string, value: any) => any): string {
-        return JSON.stringify(data, replacer, 2);
+    function stringify(data: unknown, replacer?: (key: string, value: unknown) => unknown): string {
+        return JSON.stringify(data, replacer, 2) ?? "undefined";

```

Comment:
this would have been equivalent to `... ?? ""` before, right? Do any baselines change?

Reply:
Actually, the previous code would have returned `undefined` (not string), which is why I was conservative and returned the string `"undefined"` since that's what would happen when it's cast to a string.